api/resources_portal/migrations/0001_initial.py

```diff
@@ -734,6 +742,25 @@ class Migration(migrations.Migration):
                 to="resources_portal.Material",
             ),
         ),
+        migrations.AddField(
+            model_name="user",
+            name="grants",
+            field=models.ManyToManyField(
+                through="resources_portal.GrantUserAssociation", to="resources_portal.Grant"
+            ),
+        ),
+        migrations.AddField(
+            model_name="user",
+            name="groups",
+            field=models.ManyToManyField(
+                blank=True,
+                help_text="The groups this user belongs to. A user will get all permissions granted to each of their groups.",
+                related_name="user_set",
+                related_query_name="user",
+                to="auth.Group",
+                verbose_name="groups",
+            ),
+        ),
```

Comment:
Hey so we're actually at the point where I think we should switch back to adding new migrations, because we have the staging server up and I don't really want to have to re-init its database :)

Reply:
Gotcha. I just separated it out (: